packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -385,6 +387,18 @@ class _ModalScopeStatus extends InheritedWidget {
   }
 }
 
+/// Indicates whether the current route should be popped.
+enum RoutePopDisposition {
+  /// Pop the route.
+  pop,
+
+  /// Do not pop the route.
+  doNotPop,
```

Comment:
The docs for pop and doNotPop don't follow the style guide (both could be written by someone who knows nothing except the name of the enum and the name of the value). Please add more colour to these explaining more about what they mean, e.g.: what their implications are, how you might use them, code samples, methods that use these values, how they interact with Android or with nested Navigators, etc.